src/main.py

```diff
@@ -0,0 +1,55 @@
+import asyncio
```

Comment:
rename `src` to meaningful module name like `sim_dev` or something like this
rename `main.py` to `__main__.py` - so you can call module like `python -m sim_dev -a arg1 -b arg2 ...`
Let's use [click](https://click.palletsprojects.com/en/5.x/) to create CLI - please check works of other students